AFQ/segmentation.py

```diff
@@ -218,7 +221,18 @@ def _seg_afq(self, bundle_dict, streamlines, fdata=None, fbval=None,
         "Tract Profiles of White Matter Properties: Automating Fiber-Tract
         Quantification" PloS One 7 (11): e49790.
         """
+        if img_affine is not None:
+            if (mapping is None or
+                fdata is not None or
+                fbval is not None or
+                    fbvec is not None):
```

Comment:
I think that pep8 is upset about those `or`s being at the end of lines. See: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator